### ISSUE TITLE
Why is the attenuation weight for Medium range higher than for Close range?

### ISSUE THREAD

**Issue by mthiop** [corona-warn-app/cwa-documentation#775]
Hi,

i have a question regarding the attenuation and the associated weights.

My understanding is that the exposure time is weighted based on attenuation.
Isn't the assumption that smaller attenuation pose a higher risk than larger attenuation?

According to the [risk calculation parameters](https://github.com/corona-warn-app/cwa-server/blob/main/services/distribution/src/main/resources/main-config/v2/risk-calculation-parameters-1.15.yaml) and [Figure 16 of the solution architecture,](https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md#attenuation-buckets) the weight for "Close" range is _0.8_ and the weight for "Medium" range is _1.0_.
In a previous [commit](https://github.com/corona-warn-app/cwa-server/blob/2d8ceaec87f10ce1d5521cd0fd600b2d48a0be60/services/distribution/src/main/resources/main-config/v2/risk-calculation-parameters-1.15.yaml) dated March 1, 2021, the "Close" range had a higher weight than the "Medium" range. This made sense.

Shouldn't the weight for "Close" range be higher than the "Medium" range, as it was before?

Is there a reason behind this change?

**Comment by mlenkeit** [comment]
@mthiop this was the result of the analysis/optimization that Fraunhofer Institut did, see:
- CWA Blog: [Project Team further improves Corona-Warn-App's risk calculation in response to current coronavirus situation](https://www.coronawarn.app/en/blog/2021-04-16-corona-warn-app-risk-calculation-further-improved/)
- Press release from Fraunhofer Institute: [Fraunhofer unterstützt Entwicklung der Corona-Warn-App](https://www.fraunhofer.de/de/presse/presseinformationen/2020/juni/wichtiger-beitrag-zur-eindaemmung-der-coronavirus-pandemie.html) (from 2020 but gives you a good idea of the kind of analysis they did)

**Comment by Ein-Tim** [comment]
@mthiop Are you satisfied with @mlenkeit's answer and this issue can be closed?

**Comment by OlympianRevolution** [comment]
I would actually like somebody from Fraunhofer to confirm or publish the analysis data.

**Comment by MikeMcC399** [comment]
@OlympianRevolution 
> I would actually like somebody from Fraunhofer to confirm or publish the analysis data.

The Science blog article: [About the Effectiveness and Benefits of the Corona-Warn-App](https://www.coronawarn.app/en/science/2021-06-15-science-blog-1/) in the section [What is the purpose of evaluating the CWA and what aspects play a role in the evaluation?](https://www.coronawarn.app/en/science/2021-06-15-science-blog-1/#what-is-the-purpose-of-evaluating-the-cwa-and-what-aspects-play-a-role-in-the-evaluation) includes a video showing how the calibration was carried out. It also says: "We also intend to provide more information about the results of these tests.". I don't remember seeing any further data published however.

**Comment by OlympianRevolution** [comment]
For such a patently illogical conclusion it would be good to publish the data or at least paper, to make sure this is not a typo.

**Comment by vaubaehn** [comment]
I couldn't find any explanations from a quick internet research for this special case in question here. However, there are theoretically statistical implications that would explain why the attenuation weights had been adopted in this way:
When analysis of the former attenuation weight model reveals a high sensitivity (correct detection of high risk contacts) but a poor specifity (i.e., incorrect detection of no/low risk contacts resulting in too many high risc contacts/false positives), you would refine the model (e.g., by a receiver operating curve) to better balance sensitivity and specifity. That could mean that you reduce the weights for all attenuation buckets to enhance specifity, which results in a too low sensitivity. But by again increasing the weight of the medium range attenuation bucket the result could be again a better sensitivity with marginal loss in specifity, hence a better balanced model.
Imagine a radio with an equilizer that is connected to loudspeakers that only have large bass membran and a powerful high tone unit. If you turn all equilizer frequencies to maximum, the sound may be distorted by low and high frequencies. If you then pull down the low and high frequencies of the equilizer but leave the middle frequencies untouched, the sound becomes balanced.
That is what _may_ have happened here.

But I agree to you, it would be good to have an official publication about the study.

**Comment by mlenkeit** [comment]
@OlympianRevolution @vaubaehn thank you for your interest in the science behind this! It looks like there is indeed no scientific publication about this available _yet_. I'll let you know once something is published.

I suggest keeping the issue open until then.

**Comment by mlenkeit** [comment]
@OlympianRevolution @vaubaehn in the meantime, you might find this one here interesting (if you have access): https://ieeexplore.ieee.org/document/9662591

Let's still keep the issue open. There might be something else I can refer you to, soon 😉

**Comment by OlympianRevolution** [comment]
Sadly the optimized parameters section does not go in to detail about how the optimization was performed or how increasing the close range weight would decrease the F2 score. It also does not have any discussion of the counterintuitive short range weight. I suspect there may have been relatively little data at short range. And thus the short range weight may be due to chance. But without the data or eval code it is impossible to know.

**Comment by Ein-Tim** [comment]
@mlenkeit 

Are you able to share an update with us here?

**Comment by AnonymousUserUse** [comment]
@mlenkeit

Are you able to share an update with us here now?

**Comment by mlenkeit** [comment]
@AnonymousUserUse yes and no. I have checked with the author and the document is still being finalized. I don't have an ETA though.
